assets/js/base/components/store-notices-container/store-notices-container.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Notice } from 'wordpress-components';
+import { useStoreNoticesContext } from '@woocommerce/base-context';
+import { filter } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const getWooClassName = ( { status = 'default' } ) => {
+	switch ( status ) {
+		case 'error':
+			return 'woocommerce-message woocommerce-error';
+		case 'success':
+			return 'woocommerce-message woocommerce-success';
+		case 'info':
+		case 'warning':
+			return 'woocommerce-message woocommerce-info';
+	}
+	return '';
+};
+
+const StoreNoticesContainer = ( { className, notices } ) => {
+	const { removeNotice } = useStoreNoticesContext();
+	const wrapperClass = classnames( className, 'wc-block-components-notices' );
+
+	if ( ! notices.length ) {
+		return null;
+	}
+
+	const regularNotices = filter(
```

Comment:
I know we get lodash for free, but for methods which are equivalent in raw JavaScript, I usually feel it's better to use the standard method instead of lodash's.

Reply:
Agreed. I had already replaced this but forgot to commit and push. Thanks for pointing this out though because we definitely should avoid lodash if it has a clear replacement like this :)